lib/algos/list/basic.js

```diff
@@ -1,12 +1,14 @@
 'use strict'; // eslint-disable-line strict
 
+const Extension = require('./Extension').default;
+
 const checkLimit = require('./tools').checkLimit;
 const DEFAULT_MAX_KEYS = 10000;
 
 /**
  *  Class of an extension doing the simple listing
  */
-class List {
+class List extends Extension {
```

Comment:
Is this still an issue here considering the time nodejs spends to do "super" compared to the listing process itself? The number 4x is significant but in the context of listing I do not really see that as an issue.
Definitely that needs to be improved though. What is your suggestion here: removing `super` and duplicating the code or is there any cool approach you can think of?